app/javascript/packs/ssn-field.ts

```diff
@@ -1,14 +1,13 @@
 import Cleave from 'cleave.js';
 
-/* eslint-disable no-new */
 function formatSSNFieldAndLimitLength() {
-  const inputs = document.querySelectorAll('input.ssn-toggle[type="password"]');
+  const inputs = document.querySelectorAll<HTMLInputElement>('input.ssn-toggle[type="password"]');
 
   if (inputs) {
-    [].slice.call(inputs).forEach((input) => {
-      const toggle = document.querySelector(`[aria-controls="${input.id}"]`);
+    inputs.forEach((input) => {
```

Comment:
did `inputs` have `forEach` all along? or was this something we get as part of dropping IE?

Reply:
This is possible because of dropping IE: https://caniuse.com/mdn-api_nodelist_foreach